Podfile

```diff
@@ -222,8 +222,8 @@ abstract_target 'Apps' do
 
   pod 'Gridicons', '~> 1.1.0'
 
-  pod 'WordPressAuthenticator', '~> 3.2', '>= 3.2.1'
-  # pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :branch => 'fix/showcancel'
+  pod 'WordPressAuthenticator', '~> 3.2', '>= 3.2.2-beta.1'
```

Comment:
👋 Hey @salimbraksa - I was initially thrown off by these dual version arguments ([that were added here](https://github.com/wordpress-mobile/WordPress-iOS/commit/016679ca24bdff97e94e31f6fa66e1a88f87143e)) but they appear to be valid RubyGems versioning policies.

So, I read this as:
- Version `3.2` up to but not including `3.3` and higher
- Greater than or equal to `3.2.2-beta.1`

Do I have that right @mokagio? Would the optimistic operator alone accomplish this? e.g.:

`~> 3.2.2-beta.1`

This isn't a blocker, I'm just curious to learn more.